README.md

```diff
@@ -1,2 +1,26 @@
 # pay-apps
-Dynamic Apps integrating with GOV.UK Pay   
+
+Offline apps integrating with GOV.UK Pay in Java(Dropwizard)
```

Comment:
Offline?

Reply:
not fully sure whats best to describe. Leaving it as just `apps` until we sort out the domain language next week.